gatsby_fp/src/components/fieldComponents/GenInfoTable.js

```diff
@@ -0,0 +1,121 @@
+import React from "react"
+
+export default function GenInfoTable({
+  school,
+  title,
+  imgsrc,
+  building,
+  room,
+  phone,
+  fax,
+  username,
+  website,
+}) {
+  return (
+    <>
+      <div className="table_wrapper">
+        <div className="table_wrapper_inner">
+          <table
+            cellPadding="5"
+            cellSpacing="0"
+            border="0"
+            className="bioregion"
+            width="100%"
+          >
+            <tbody>
+              <tr>
+                <td valign="top" align="left" width="160px">
+                  <img
+                    src={`data:image/jpeg;base64,${imgsrc}`}
+                    className="bioimage"
+                    alt=""
```

Comment:
For accessibility, we populate alt tags except when the image is purely decorative. In this case, something like `Photo of {pf_first_name} {pf_last_name}` would be great (you'll need to pass them in as additional props).